wsgi.py

```diff
@@ -6,255 +6,348 @@
 import re
 import os
 from flask import Flask, redirect, url_for, request, Response
-#from flask_restful import reqparse
+from flask_httpauth import HTTPBasicAuth
+
+# from flask_restful import reqparse
 
 import sys
 
-from openshift_rolebindings import *
-from openshift_project import *
-from openshift_identity import *
-from openshift_user import *
+from openshift import *
 
 application = Flask(__name__)
+auth = HTTPBasicAuth()
 
-if __name__ != '__main__':
-    gunicorn_logger = logging.getLogger('gunicorn.error')
+if __name__ != "__main__":
+    gunicorn_logger = logging.getLogger("gunicorn.error")
     application.logger.handlers = gunicorn_logger.handlers
     application.logger.setLevel(gunicorn_logger.level)
 
 
-def get_user_token():
-    with open('/var/run/secrets/kubernetes.io/serviceaccount/token', 'r') as file:
+def get_openshift():
+    version = os.environ["OPENSHIFT_VERSION"]
+    url = os.environ["OPENSHIFT_URL"]
+    if version == 3:
```

Comment:
This will never match; environment variables always have string values. You want:

```
if version == '3':
```